code/game/machinery/Sleeper.dm

```diff
@@ -262,6 +262,10 @@
 		stop_processing() //Shut down; stasis off, filtering off, stop processing.
 		return
 
+	//Life support
+	if(occupant)
+		occupant.adjustOxyLoss(-occupant.getOxyLoss()) // keep them breathing, pretend they get IV dexalinplus
```

Comment:
You can substitute this check with:
`occupant?.adjustOxyLoss(-occupant.getOxyLoss())`